docs/reference/commandline/dockerd.md

```diff
@@ -1279,3 +1282,25 @@ $ sudo dockerd \
         --data-root=/var/lib/docker-bootstrap \
         --exec-root=/var/run/docker-bootstrap
 ```
+
+### Default network options
+
+The `--default-network-opt` flag, and the equivalent `default-network-opts` key
+in the configuratio file, lets you specify custom default options for custom
+networks.
+
+This option accepts a value with the format `driver=opt=value`, for example:
```

Comment:
It may be good to mention that these options are driver-specific, and perhaps we should cross-reference where to find those options.

For the bridge driver, there's a table here; https://docs.docker.com/network/drivers/bridge/#options, but I haven't checked if there's a similar list for all of the other drivers, and what a good page would be to link to.

We could link to the overview page of networking;

```markdown
Available network options are specific to the network-driver. Refer to the 
[network drivers section](https://docs.docker.com/network/drivers/) to learn
about available options.
```

But that would still require users to try to find options for each driver (perhaps that's fine?), or we could of add a link to this specific example;

```markdown
This example uses the "bridge" network driver. Refer to the 
[bridge network driver page](https://docs.docker.com/network/drivers/bridge/#options)
for an overview of available driver options.
```

Reply:
afaict, only the bridge and macvlan driver actually seem to have options? At least those are the only docs pages that list any options. Is that right @akerouanton ?